lutris/gui/config/boxes.py

```diff
@@ -423,10 +423,15 @@ def generate_file_chooser(self, option, path=None):
         file_chooser.entry.connect("changed", self._on_chooser_file_set, option_name)
 
     def _on_chooser_file_set(self, entry, option):
-        """Action triggered on file select dialog 'file-set' signal."""
-        if not os.path.isabs(entry.get_text()):
-            entry.set_text(os.path.expanduser(entry.get_text()))
-        self.option_changed(entry.get_parent(), option, entry.get_text())
+        """Action triggered when the field's content changes."""
+        text = entry.get_text()
+        if text.startswith('file:///'):
```

Comment:
Note that there are three slashes. I wanted to avoid a situation where user types "file://" (for whatever reason), only to guess why Lutris had wiped all this text. With one slash remaining after the wipe, it is less confusing.